providers/configcat/go.mod

```diff
@@ -0,0 +1,14 @@
+module github.com/open-feature/go-sdk-contrib/providers/configcat
+
+go 1.13
```

Comment:
Minor - I hope this is to be compatible with configcat go-sdk v7 and to support old platforms?

Reply:
That's correct I noticed [a comment from you](https://github.com/open-feature/go-sdk-contrib/pull/128#discussion_r1128585782) that seemed to suggest it was best to align with the underlying SDK.

Should this instead align with [go1.18](https://github.com/open-feature/go-sdk/blob/d93f58bfb9ea92fbcbb0faa7cd7880bb629a6ebe/go.mod#L3) from the `go-sdk`?